test/analysis/minimize_metabolic_adjustment.jl

```diff
@@ -1,11 +1,7 @@
 @testset "MOMA" begin
-    model = load_model(StandardModel, model_paths["e_coli_core.json"])
+    model = test_toyModel()
 
-    sol = parsimonious_flux_balance_analysis_dict(
-        model,
-        OSQP.Optimizer;
-        modifications = [silence, change_optimizer_attribute("polish", true)],
-    )
+    sol = [looks_like_biomass_reaction(rid) ? 0.5 : 0.0 for rid in reactions(model)]
```

Comment:
just write the vector manually here (this is going to be zero vector because of no biomass). I'd go with something like `collect(1:n_reactions(model))`

Reply:
actually this does pick up the biomass functions!

Reply:
but yeah, it is only 7 items long... I could just write it out :)

Reply:
actually, its a dict.. so it will take up even more space if I write it out by hand. It looks like `this:
```julia
julia>sol = [looks_like_biomass_reaction(rid) ? 0.5 : 0.0 for rid in reactions(model)]
7-element Vector{Float64}:
 0.0
 0.0
 0.0
 0.0
 0.0
 0.5
 0.5
```

Reply:
ah it has names, good. :D